gooddata-pandas/gooddata_pandas/data_access.py

```diff
@@ -42,13 +42,16 @@ def __init__(self, columns: ColumnsDef, index_by: Optional[IndexDef] = None) ->
         Args:
             columns (ColumnsDef): Input columns to process and build internal mappings.
             index_by (Optional[IndexDef], optional): Index definition to process. Defaults to None.
+            is_cancellable (Optional[bool]): Whether the execution of this definition should be cancelled when
+                the connection is cancelled.
```

Comment:
small stuff with wording; the "connection is interrupted" instead of "connection is cancelled". connection cancellation is not a commonly used term i believe

Reply:
good point, reworded